.travis.yml

```diff
@@ -1,39 +1,29 @@
 ---
 
 sudo: required
+language: minimal
 services:
   - docker
 
 
-env:
-  global:
-    - image=flaconi/prometheus-jmx-exporter
-
-install:
-  # Get newer docker version
-  - max=100; i=0; while [ $i -lt $max ]; do if sudo apt-get update; then break; else i=$((i+1)); fi done
-  - max=100; i=0; while [ $i -lt $max ]; do if sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce; then break; else i=$((i+1)); fi done
-  - docker version
-
-
 ###
 ### Build Docker image
 ###
 before_script:
-  - docker build -t "${image}" .
+  - make build
 
 
 ###
 ### Push Docker image
 ###
 script:
   - if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
-      echo "${DOCKERHUB_PASSWORD}" | docker login --username "${DOCKERHUB_USERNAME}" --password-stdin &&
+      make login USER="${DOCKERHUB_USERNAME}"  PASS="${DOCKERHUB_PASSWORD}";
       if [ "${TRAVIS_BRANCH}" == "master" ]; then
-        docker push "${image}:latest";
+        make push;
```

Comment:
for my information, how does `make push` get the $(TAG)

Reply:
default is `latest`

Reply:
I see it, thanks.